src/StreamMaster.h

```diff
@@ -28,7 +28,6 @@ namespace FileWriter {
 template <typename Streamer> class StreamMaster {
   using StreamerStatus = Status::StreamerStatus;
   using StreamMasterError = Status::StreamMasterError;
-  friend class CommandHandler;
```

Comment:
👍 Extra points for the funny commit message.

Reply:
👍